src/main/java/uk/gov/hmcts/reform/orgrolemapping/monitoring/service/ProcessEventTracker.java

```diff
@@ -37,6 +38,8 @@ private Map<String, String> createPropertiesMap(ProcessMonitorDto processMonitor
         properties.put("Id", processMonitorDto.getId().toString());
         properties.put("ProcessType", processMonitorDto.getProcessType());
         properties.put("StartTime", processMonitorDto.getStartTime().toString());
+        properties.put("ProcessSteps", processMonitorDto.getProcessSteps().stream().map(Object::toString)
```

Comment:
NB: There is a limit to the [Property value string length](https://github.com/MicrosoftDocs/azure-docs/blob/main/includes/application-insights-limits.md) in azure.  

Although the configurable `application.yaml` value `"professional.refdata.pageSize"` allows us to control the max size of each page loaded, i.e. number of orgs in each step in process 3.  Given the step description contains a CSV list of the orgIds is there a risk we will breach the limit off a single value: once all the steps are joined together into one entry?

Reply:
I've added some limitations on the fields. There is a risk of exceeding the limit, we may have to review this and optimise as we move along